src/pyosmeta/file_io.py

```diff
@@ -21,124 +20,158 @@ def get_api_token() -> str:
     return os.environ["GITHUB_TOKEN"]
 
 
-@dataclass
-class YamlIO:
+def load_pickle(filename):
+    """Opens a pickle"""
+    with open(filename, "rb") as f:
+        return pickle.load(f)
+
+
+def _list_to_dict(a_list: list, a_key: str) -> dict:
+    """Takes a yaml file opened and turns into a dictionary
+    The dict structure is key (gh_username) and then a dictionary
+    containing all information for the username
+
+    a_list : list
+        A list of dictionary objects returned from load website yaml
+    a_key : str
+        A string representing the dict key to use as the main key to call
+        each sub dict in the object.
+
+    """
+    final_dict = {}
+    for dict in a_list:
+        # All github usernames are lower to make this a key
+        final_dict[dict[a_key].lower()] = dict
+
+    return final_dict
+
+
+def load_website_yml(a_key: str, a_url: str):
```

Comment:
```suggestion
def load_website_yml(key: str, url: str):
```

it is not common to have an `article` as part of the variable name

Reply:
also, if it is possible, annotate the type for the output, for example:

`def load_website_yml(key: str, url: str) -> Dict[THE_KEY_TYPE, THE_VALUE_TYPE]:`

Reply:
i'm curious about this one. for me it just makes it easier (for me) to read / understand. particularly because keys() is a method. is this a python pep type standard or just not a normal way that people create these types of variables? i can see how a_url might be unnecessary as it's not related to python and explicit enough.